### PR TITLE
Adds scroll-into-view

### PR DESCRIPTION
**Pull request by dmiddle2000lb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "ng-annotate": "1.2.1"
   },
   "peerDependencies": {
-    "angular": ">=1.5"
+    "angular": ">=1.5",
+    "jquery": "2.2.3"
```
Comment (digitil) [inline package.json]: this exact version? :suspect:

File: scroll-into-view/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ *
+ * @description
+ *
+ * If this directive's property evalutes to true, the element will be scrolled into view.
+ *
+ * The speed can be set with the attribute `bd-scroll-into-view-speed`
+ *
+ * @depends buildium.common.services.page
+ *
+ * <!-- example #1 -->
```
Comment (digitil) [inline scroll-into-view/index.js]: use the `@example` tag for documentation generation

File: scroll-into-view/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+let $ = require('jquery');
+
+/**
+ * @ngdoc directive
+ * @name buildium.common.bdScrollIntoView
+ *
+ * @restrict 'A'
```
Comment (digitil) [inline scroll-into-view/index.js]: no quotes around the A

File: scroll-into-view/index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ *
+ * @description
+ *
+ * If this directive's property evalutes to true, the element will be scrolled into view.
+ *
+ * The speed can be set with the attribute `bd-scroll-into-view-speed`
+ *
+ * @depends buildium.common.services.page
```
Comment (digitil) [inline scroll-into-view/index.js]: use `@requires`

File: scroll-into-view/index.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+ * <!-- /example -->
+ *
+ * <!-- example #2: a very sloooow scroll -->
+ *    <input bd-scroll-into-view="vm.someVariable" bd-scroll-into-view-speed="1500" />
+ * <!-- /example -->
+ */
+
+function scrollToElement(element, fn, duration) {
+    let target = element.length ? element : $(element);
+    let callback = (fn && typeof fn === 'function') ? fn : function emptyFn() {};
```
Comment (digitil) [inline scroll-into-view/index.js:28]: just the `typeof` check would be enough for the condition here right? since a function is truthy

**Review by digitil — APPROVED** [review]

**Review by ryan-pierce-buildium — APPROVED** [review]
wheeeeee

File: package.json [hunk 6]
```diff
@@ -22,6 +22,7 @@
     "ng-annotate": "1.2.1"
   },
   "peerDependencies": {
-    "angular": ">=1.5"
+    "angular": ">=1.5",
+    "jquery": "^2.2.3"
```
Comment (jweisbeck) [inline package.json]: I have this set to >=2.0 on a different branch

**Review by jweisbeck — APPROVED** [review]

**Pull request merged**